plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/preferences/CloudSdkPreferenceArea.java

```diff
@@ -86,15 +90,19 @@ public void widgetSelected(SelectionEvent event) {
         WorkbenchUtil.openInBrowser(PlatformUI.getWorkbench(), event.text);
       }
     });
-
+    
+    sdkVersionLabel = new Label(parent, SWT.LEFT);
+    sdkVersionLabel.setText(Messages.getString("SdkVersion", "Unset"));
```

Comment:
We need to explicitly set the font size here due to what we're doing in line 86 as the label's font is one pixel too small:
<img width="183" alt="screen shot 2018-02-15 at 9 14 01 pm" src="https://user-images.githubusercontent.com/202851/36290794-38ce4f2c-1295-11e8-92fc-1cd6343c376d.PNG">

Reply:
done